docs/content/preview/yugabyte-platform/prepare/networking.md

```diff
@@ -22,9 +22,9 @@ The following ports need to be open. (The default port numbers can be customized
 
 | From | To | Requirements |
 | :--- | :--- | :--- |
-| DB&nbsp;nodes | DB&nbsp;nodes | Open the following ports for communication between nodes in clusters. They do not need to be exposed to your application.<ul><li>7000 - YB-Master HTTP endpoint</li><li>7100 - YB-Master RPC</li><li>9000 - YB-TServer HTTP endpoint</li><li>9100 - YB-TServer RPC</li><li>18018 - YB Controller</li></ul> |
-| YBA&nbsp;node | DB nodes | Open the following ports on database cluster nodes so that YugabyteDB Anywhere can provision them.<ul><li>22 - SSH</li><li>5433 - YSQL server</li><li>7000/7100 - YB-Master HTTP/RPC endpoint</li><li>9000/9100 - YB-TServer HTTP/RPC endpoint</li><li>9042 - YCQL server</li><li>9070 - Node agent</li><li>9300 - Prometheus Node Exporter</li><li>12000 - YCQL API</li><li>13000 - YSQL API</li><li>18018 - YB Controller</li></ul>SSH is not required after initial setup and configuration, but is recommended for subsequent troubleshooting. If you disallow SSH entirely, you must manually set up each DB node (see [Provisioning on-premises nodes](../server-nodes-software/software-on-prem-manual/)). |
-| Application | DB nodes | Open the following ports on database cluster nodes so that applications can connect via APIs.<ul><li>5433 - YSQL server</li><li>9042 - YCQL server</li></ul> |
+| DB&nbsp;nodes | DB&nbsp;nodes | Open the following ports for communication between nodes in clusters. They do not need to be exposed to your application. For universes with [Node-to-Node encryption in transit](../../security/enable-encryption-in-transit/), communication over these ports is encrypted.<ul><li>7000 - YB-Master HTTP(S)</li><li>7100 - YB-Master RPC</li><li>9000 - YB-TServer HTTP(S)</li><li>9100 - YB-TServer RPC</li><li>18018 - YB Controller RPC</li></ul> |
+| YBA&nbsp;node | DB nodes | Open the following ports on database cluster nodes so that YugabyteDB Anywhere can provision them.<ul><li>22 - SSH</li><li>5433 - YSQL server</li><li>7000/7100 - YB-Master HTTP/RPC</li><li>9000/9100 - YB-TServer HTTP/RPC</li><li>9042 - YCQL server</li><li>9070 - Node agent RPC</li><li>9300 - Prometheus Node Exporter HTTP</li><li>12000 - YCQL API</li><li>13000 - YSQL API</li><li>18018 - YB Controller RPC</li></ul>SSH is not required after initial setup and configuration, but is recommended for subsequent troubleshooting. If you disallow SSH entirely, you must manually set up each DB node (see [Provisioning on-premises nodes](../server-nodes-software/software-on-prem-manual/)). |
```

Comment:
Some of the YBA -> DB node traffic is also possible to encrypt or always encrypted. I think adding these details in the prepare page is the wrong place to do this. Can we add this to the encryption / security section instead?